weblate/templates/translation.html

```diff
@@ -312,24 +319,78 @@ <h4 class="panel-title">
 {% endif %}
 
 <div class="tab-pane" id="download">
-<form action="{% url 'download_translation' project=object.component.project.slug component=object.component.slug lang=object.language.code %}" method="get" data-persist="download-translation">
-<div class="panel panel-default">
-<div class="panel-heading"><h4 class="panel-title">{% trans "Customize download" %}</h4></div>
-<div class="panel-body">
-{% crispy download_form %}
-</div>
-<div class="panel-footer">
-<input type="submit" value="{% trans "Download" %}" class="btn btn-primary" />
-</div>
-</form>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4 class="panel-title">
+        {% documentation_icon 'user/files' 'download' right=True %}
+        {% trans "Quick downloads" %}
+      </h4>
+    </div>
+    <table class="table table-listing table-striped">
+      <tr>
+        <th class="number">{{ object.stats.all|intcomma }}</th>
+        <td>
+          {% trans "Original format as translated in the repository" %}
```

Comment:
```suggestion
          {% trans "Original format as stored in the repository" %}
```

Reply:
Let’s keep translated. Not ideal from the grammar view, but better describe what it does.

Reply:
As you wish, but sounds odd...

Reply:
Comma after translated? Is it another semicolon?

Reply:
@Fat-Zer I see, but don’t have a better formulation for now.

@comradekingu This is outdated, probably. 😄